### PR TITLE
Document recently added byte string related builtins

### PR DESCRIPTION
**Pull request by vaivaswatha**
References:
- https://github.com/Zilliqa/scilla/commit/33b519e9ef8eaf5172bdc2be9e5603788379204a
- https://github.com/Zilliqa/scilla/commit/9d298f6760e1b1b7f0e2519f110736c0d7cf83c7
- https://github.com/Zilliqa/scilla/commit/1fe27177d9b325679f53aa8ea112bfe5ec929f48
- https://github.com/Zilliqa/scilla/commit/accce15e80484e01ba6c955b1881fc80fd3eb826
- https://github.com/Zilliqa/scilla/commit/ff58176f974832e9b5e80d019ee8518598527aa5
- https://github.com/Zilliqa/scilla/commit/591c728f2f396beb4faae997dae423ceb5672a74

### CODE REVIEW

File: docs/source/scilla-in-depth.rst [hunk 1]
```diff
@@ -762,12 +767,27 @@ below, ``Any`` can be of type ``IntX``, ``UintX``, ``String``, ``ByStr20`` or
...
 
-- ``builtin to_bystr h`` : Convert a hash ``h`` of type ``ByStrX`` (for
+- ``builtin to_bystr h`` : Convert a value ``h`` of type ``ByStrX`` (for
   some known ``X``) to one of arbitrary length of type ``ByStr``.
 
-- ``builtin to_uint256 h`` : Convert a hash ``h`` to the equivalent
-  value of type ``Uint256``. ``h`` must be of type ``ByStrX`` for some
-  known ``X`` less than or equal to 32.
+- ``builtin substr h idx len`` : Extract the sub-byte-string of ``h`` of
+  length ``len`` starting from position ``idx``. ``idx`` and
```
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:774]: This sentence ends abruptly

**Review by anton-trunov — APPROVED** [review]
LGTM overall!

File: docs/source/scilla-in-depth.rst (hunk 1 above)
Comment (vaivaswatha) [reply docs/source/scilla-in-depth.rst:774]: > This sentence ends abruptly

Thank you, Fixed.

Comment (anton-trunov) [reply docs/source/scilla-in-depth.rst:774]: Cool!

**Pull request merged**